### PR TITLE
Update clone command

### PR DESCRIPTION
**Pull request by quevon24**
I made some changes to the clone command to warn the user and finish execution if CL_API_TOKEN is not set.

It now also checks that the API token is valid and finish execution immediately if it is not.

And finally i updated the docstrings.

### CODE REVIEW

File: cl/scrapers/management/commands/clone_from_cl.py [hunk 1]
```diff
@@ -80,10 +81,31 @@ def __init__(self, message: str) -> None:
 
 
 def get_id_from_url(api_url: str) -> str:
-    """Get the PK from an API url"""
+    """Get the PK from an API url
+
+    :param api_url: api url with a pk
+    :return: list of opinion cluster objects
```
Comment (mlissner) [inline cl/scrapers/management/commands/clone_from_cl.py:87]: This doesn't seem right, is it?

**Review by mlissner — APPROVED** [review]
One little thing, then let's merge!

File: cl/scrapers/management/commands/clone_from_cl.py (hunk 1 above)
Comment (quevon24) [reply cl/scrapers/management/commands/clone_from_cl.py]: It's already fixed and merged

Comment (mlissner) [reply cl/scrapers/management/commands/clone_from_cl.py]: Thank you Kevin! I appreciate your continued enhancements to this script. It seems like it's becoming a popular one!

**Pull request merged**